blog-service/2023-09-08-search.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Query Timer Enhancement (Search)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - search
+  - query
+  - elapsed timer
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Bug fix - We've enhanced the elapsed query timer so that while a query is running and your users navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.
```

Comment:
```suggestion
Bug fix - We've enhanced the elapsed query timer so that while a query is running, and your users navigate to a different Sumo internal tab or browser tab and then return, the timer no longer resets to zero seconds but instead displays the precise amount of time that has passed.
```